.gitignore

```diff
@@ -13,3 +13,13 @@
 
 # This is used for website deployment
 site.tar
+
+# ignore html files in directories except for build
+develop/*html
+use/*html
+install/*html
+
+# ignore derivative directories and files
+develop/*_file/
+use/*_file/
+install/*_file/
```

Comment:
It's plural, not singular. At least for me when I render. I.e., `*_files`, not `*_file`.